synchronization/src/main/java/de/cyface/synchronization/Http.java

```diff
@@ -63,10 +65,11 @@ HttpsURLConnection openHttpConnection(@NonNull URL url, @NonNull SSLContext sslC
      * @throws ResponseParsingException When the http response could not be parsed.
      * @throws SynchronisationException When the new data output for the http connection failed to be created.
      * @throws UnauthorizedException If the credentials for the cyface server are wrong.
+     * @throws NetworkErrorException when the connection's input or error stream was null
      */
     HttpResponse post(HttpURLConnection connection, JSONObject payload, boolean compress)
             throws RequestParsingException, DataTransmissionException, SynchronisationException,
-            ResponseParsingException, UnauthorizedException, BadRequestException;
+            ResponseParsingException, UnauthorizedException, BadRequestException, NetworkErrorException;
```

Comment:
Is this also seen as an API change? I guess so ... thus the code version bump

Reply:
Correct. Checked Exceptions are always an API change.